content/blog/2024-09-06-tmix-08/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "This Month in Xilem, August 2024"
+authors = ["Daniel McNab"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+<figure>
+
+<img style="height: auto; margin: 0 auto" src="../../linebender.svg" alt="Linebender logo; smoothly stroked overlapping lowercase letters, an l and a b." height="200" width = "200">
+
+<figcaption>
+
+Linebender has a new logo, which was contributed by [Markus Siglreithmaier](https://github.com/msiglreith).
+We're seeking feedback on Zulip in [#linebender>logo](https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/logo).
+
+</figcaption>
+</figure>
+
+## Xilem
+
+Xilem is our flagship GUI project, inspired by SwiftUI.
+
+- We integrated basic animated variable font rendering ([xilem#507][]).
+- Philipp Mildenberger moved us back to using `Viewmarker` in [xilem#472][].
+  This restores the ability to return `ViewSequence`s, by making a tradeoff to work around Rust's faulty trait ambiguity rules.
+- We added Views for some existing and new widgets:
+  - spinners, useful for showing indetermine progress ([xilem#497][]);
+  - progress bars ([xilem#513][]); and
+  - the `Portal` widget, which brings support for scroll areas to Xilem ([xilem#561][]).
+- Xilem Web:
+    <!-- TODO: -->
```

Comment:
Sorry, I'm quite busy right now.
I would say that not too much has changed. But maybe towards the end of the year we can do a detailed post with examples, joys and pains?

Reply:
Hmm. There were quite a few fixes, mostly related to the `AnyView`, not sure if this is worth mentioning.

Apart from this the following *may* or *may not* be noteworthy:

* [xilem#486](https://github.com/linebender/xilem/pull/486) - `interval(ms, callback)` view
* [xilem#493](https://github.com/linebender/xilem/pull/493) - Add support for opacity in `svg::Fill` and dashes, opacity in `svg::Stroke` views.
* [xilem#494](https://github.com/linebender/xilem/pull/494) - Change `events::OnResize` to be a `ResizeObserver` to observe targeted resize changes on the element this view composes.
* [xilem#482](https://github.com/linebender/xilem/pull/482) - Allow `DomFragment` instead of `DomView` as root (i.e. allow `0..N` instead of `1` view(s))
* [xilem#495](https://github.com/linebender/xilem/pull/495) - Basic hydration support, used by `Templated` view to make often used (bigger) views more efficient.
* [xilem#517](https://github.com/linebender/xilem/pull/495) - Basically (finish) document everything in the public API (or at least make it *fairly* complete)


Yeah as @flosse says, I think we should (as well for xilem itself) add a few more examples and info how everything is supposed to be used, like a step-by-step tutorial, but this is orthogonal to this PR/TMIX. So that it makes sense to go more public, but I think first, we may want a clearer story on SSR and related.